casepro/msgs/forms.py

```diff
@@ -79,3 +79,65 @@ def clean_keywords(self):
     class Meta:
         model = Label
         fields = ('name', 'description', 'is_synced', 'keywords', 'groups', 'field_test', 'ignore_single_words')
+
+
+class LanguageForm(forms.ModelForm):
+
+    code = forms.CharField(label=_("Code"), max_length=6,
+                           widget=forms.TextInput(attrs={'placeholder': 'eng_UK'}))
+    name = forms.CharField(label=_("Name"), max_length=100, required=False,
+                           widget=forms.TextInput(attrs={'placeholder': 'English'}))
+    location = forms.CharField(label=_("Location"), max_length=100, required=False,
+                               widget=forms.TextInput(attrs={'placeholder': 'United Kingdom'}))
+
+    def clean_code(self):
+        code = self.cleaned_data['code'].strip()
+        normalized_code = normalize(code)
+
+        if len(normalized_code) != 6:
+            raise forms.ValidationError(_("Language codes should be 6 characters long"))
+
+        if normalized_code[3] != normalize('_'):
+            raise forms.ValidationError(_("Language name and location should be seperated by an underscore"))
+
+        return normalize_language_code(normalized_code)
+
+    class Meta:
+        model = Language
+        fields = ('code', 'name', 'location')
+
+
+class FaqForm(forms.ModelForm):
+
+    question = forms.CharField(label=_("Question"), max_length=140)
+    answer = forms.CharField(label=_("Answer"), max_length=140)
+    language = forms.ModelChoiceField(queryset=Language.objects.filter())
```

Comment:
Needs to be org-aware, i.e. only allow selecting items in the current org. Same for `parent` and `labels`. Can do this by passing org as form kwarg, and initialising fields in __init__